### PR TITLE
fix: empty screen arrow not fully visible - WPB-11948

### PR DESCRIPTION
**Pull request by caldrian**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-11948" title="WPB-11948" target="_blank">BugWPB-11948</a>  [iOS] Empty screen arrow not fully visible
  </td></table>
  <br />
 

### Issue

The top constraint of the empty placeholder view was not set to the safe area layout guide.
This PR fixes it.

---

### Checklist

- [x] Title contains a reference JIRA issue number like `[WPB-XXX]`.
- [x] Description is filled and free of optional paragraphs.
- [x] Adds/updates automated tests.

---

### UI accessibility checklist

_If your PR includes UI changes, please utilize this checklist:_
- [x] Make sure you use the API for UI elements that support large fonts.
- [x] All colors are taken from WireDesign.ColorTheme or constructed using WireDesign.BaseColorPalette.
- [x] New UI elements have Accessibility strings for VoiceOver.

### CODE REVIEW

File: WireAnalytics/Package@swift-5.10.swift [hunk 1]
```diff
@@ -1,76 +0,0 @@
-// swift-tools-version: 5.10
-
-import Foundation
-import PackageDescription
-
-let package = Package(
```
Comment (KaterinaWire) [inline WireAnalytics/Package@swift-5.10.swift]: removing this file does not seem to be part of this PR 🤔

Comment (caldrian) [reply WireAnalytics/Package@swift-5.10.swift]: fixed, thanks

File: wire-ios/Wire-iOS Tests/ReferenceImages/ConversationListViewControllerSnapshotTests/testForShowingConversationsFilteredByGroups.1.png
Comment (caldrian) [inline wire-ios/Wire-iOS Tests/ReferenceImages/ConversationListViewControllerSnapshotTests/testForShowingConversationsFilteredByGroups.1.png:1]: With the default approach of rendering just the view the layout breaks for different tests.
Therefore I set the view controller as `rootViewController` in a window and rendered it into an UIImage instance. Later that image is passed into the SnapshotHelper.
That's also why outside the safe area we have blank space.

**Review by KaterinaWire — APPROVED** [review]

**Review by agisilaos — APPROVED** [review]
